StringThings.UnitTests/Extensions/TakeFirstStringExtensionsTestFixture.cs

```diff
@@ -0,0 +1,44 @@
+﻿using FluentAssertions;
+using System;
+using Xunit;
+using StringThings.Extensions;
+
+namespace StringThings.UnitTests.Extensions
+{
+    public class TakeFirstStringExtensionsTestFixture
```

Comment:
beautiful tests! did they feel useful to you??

Reply:
They have their purpose. For this situation where you are making a reusable public library, good testing is essential and this type of extension lends itself to simple unit tests.